<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBaseDeDonnees</class>
 <widget class="QWidget" name="WidgetBaseDeDonnees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetBaseDeDonnees</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Donnez un numéro de département</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNomDepartement">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Avoir le nom du département</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumeroDepartement">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomDepartement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du département</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRegions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choissisez une région</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxDepartements">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choisissez un département</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxVilles">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choisissez une ville</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
